hack/deploy-hub-local/lab-dns.sh

```diff
@@ -69,7 +69,7 @@ EOF
     echo "127.0.0.1 localhost.localdomain localhost" > /etc/hosts
 }
 
-function set_dnsmasq(){
+function set_dnsmasq_hub(){
     echo ">> Configuring dnsmasq"
     echo "user=dnsmasq
 group=dnsmasq
```

Comment:
@jparrill in the line 77 (not this line) bind-interfaces in systemd based host is failing avoiding getting up the service.  I think the common configuration is bind-dynamic instead of bind-interfaces. WDYT???